addon/lib/shooter.js

```diff
@@ -104,6 +105,8 @@ function extractWorker(tab, timeLimit) {
     });
     watchWorker(worker);
     worker.port.on("data", function (data) {
+      let timeFinished = new Date();
+      sendTiming("addon", "make-shot", timeFinished - timeStarted);
```

Comment:
This isn't a good name – the shot is really done by make-static-html, this is extracting metadata.  So `"extract-data"` would be a better description.